src/controllers/authControllers.ts

```diff
@@ -4,16 +4,23 @@ import {
 	UserCredential,
 } from 'firebase/auth';
 import admin from 'firebase-admin';
+import dotenv from 'dotenv';
 import { Request, Response } from 'express';
 import app from '../config/firebase.js';
 import { responses } from '../constants/responses.js';
 import { errorCodes, statusCodes } from '../constants/codes.js';
 import { Roles } from '../constants/roles.js';
+import { db } from '../index.js';
+import Collections from '../constants/collections.js';
+
+dotenv.config();
 
 export const signup = async(req: Request, res: Response): Promise<Response> => {
 	try {
 		const { displayName, email, password } = req.body;
-		const role = Roles.user;
+
+		//temporary
```

Comment:
Would be better if you add TODO comment with explanation what you or someone will need to do here

Reply:
ok, got it !